### DISCUSSION TITLE
When an MRF exists, but the Plan ID is not linked

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#738] [Q&A] [answered]
To give a specific example, a TPA leases a network from Cigna. Cigna publishes the MRF for that network, but puts a dummy plan_id value in their table of contents file: https://www.cigna.com/legal/compliance/machine-readable-files

Since there's no way to determine which MRF actually goes with that plan using EIN or HIOS, is it accurate to say that these plans are not in compliance?  Or to ask it another way, will the TPA be required to publish a ToC containing the plan_id and pointing to the Cigna MRF?

**Comment by JR-CMS** [comment, marked as answer]
If you believe you have observed an instance of potential non-compliance with the Transparency in Coverage final rules, please submit it here: https://www.cms.gov/healthplan-price-transparency/contact-us

**Reply by ghost** [reply]
That doesn't really answer my question. I'm trying to help clients be compliant with your requirements. Are you saying you don't know if the situation I described is compliant or not?

**Reply by JR-CMS** [reply]
HHS does not publicly comment on compliance matters. Using our contact/complaint form enables us to gather critical information to investigate potential non-compliance, track complaints, and assist plans, developers, and consumers directly.

**Reply by ghost** [reply]
Thank you for that explanation.

**Comment by tdfow** [comment]
The Plan ID should represent the HIOS value (10 or 5) or the Employer Identification Number (EIN).   Originally, the HIOS could be 14, 10 or 5 but guidance then updated HIOS to 10 or 5.



<html>
<body>

plan_id | Plan ID | String | The 10-digit Health Insurance Oversight System (HIOS) identifier, or, if the 10-digit HIOS identifier is not available, the 5-digit HIOS identifier, or if no HIOS identifier is available, the Employer Identification Number (EIN)for each plan or coverage offered by a plan or issuer.



</body>
</html>
